### ISSUE TITLE
SatSearchError: message: Forbidden

### ISSUE THREAD

**Issue by salvis2** [sat-utils/sat-search#95]
Pangeo maintains a Landsat 8 tutorial as part of the pangeo-tutorial notebooks. I've been going through and doing some updating and cleaning but not changing the overall tutorial content. I was pretty surprised yesterday because there was a cell that went from working normally to giving me a 403 error. Here is the cell:

```
properties =  ["landsat:tier=T1"] 
bbox = (gfa.bounds.minx[0], gfa.bounds.miny[0], gfa.bounds.maxx[0], gfa.bounds.maxy[0]) #(west, south, east, north) 
results = satsearch.Search.search(collection='landsat-8-l1', 
                        bbox=bbox, 
                        sort=['<datetime'], #earliest scene first
                        property=properties)
print('%s items' % results.found())
items = results.items()
items.save('set.geojson')
```

And here is the error:

```
SatSearchError                            Traceback (most recent call last)
<ipython-input-5-774276f8fe53> in <module>
      5                         sort=['<datetime'], #earliest scene first
      6                         property=properties)
----> 7 print('%s items' % results.found())
      8 items = results.items()
      9 items.save('set.geojson')

/srv/conda/envs/notebook/lib/python3.7/site-packages/satsearch/search.py in found(self)
     73         }
     74         kwargs.update(self.kwargs)
---> 75         results = self.query(**kwargs)
     76         return results['meta']['found']
     77 

/srv/conda/envs/notebook/lib/python3.7/site-packages/satsearch/search.py in query(cls, url, **kwargs)
     83         # API error
     84         if response.status_code != 200:
---> 85             raise SatSearchError(response.text)
     86         return response.json()
     87 

SatSearchError: {"message":"Forbidden"}
```

This code worked previously, so I'm unsure how to troubleshoot this. Any thoughts?

**Comment by chaitanya2ivvala** [comment]
Hey, I had a similar problem refer the below link it had a solution as well. 
https://github.com/sat-utils/sat-search/issues/94

**Comment by matthewhanson** [comment]
Hi @salvis2 , sorry about that, as @chaitanya2ivvala pointed out the solution is in #94 - add an environment variable with the actual URL.

I'll be adding Landsat into the new catalog, at which point the tutorials can be updated to use the new sat-search and new catalog, there's only a couple modifications that are needed.

**Comment by salvis2** [comment]
Gotcha. Thank you @chaitanya2ivvala , I forgot to check the closed issues before posting. 

@matthewhanson , do you know if you are going to release a new version of `sat-search` soon? My current use-case is a tutorial being run by binder. I think I'm pinned to v0.2.2 whenever that was put into the Docker image, so I the fix isn't working for me probably because the `os.getenv` call isn't in my `sat-search`.

**Comment by matthewhanson** [comment]
@salvis2 I posted in the above issue, but the fix for now is to pin sat-search to 0.2.3.

When the new earth search has Landsat in it we can update the notebook to use that with the new 0.3.x sat-search
